src/Analyzers/CSharp/CodeFixes/UseLocalFunction/CSharpUseLocalFunctionCodeFixProvider.cs

```diff
@@ -93,7 +93,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             {
 #if CODE_STYLE
                 var info = new CSharpCodeGenerationContextInfo(
-                    CodeGenerationContext.Default, CSharpCodeGenerationOptions.Default, new CSharpCodeGenerationService(document.Project.Services), root.SyntaxTree.Options.LanguageVersion());
+                    CodeGenerationContext.Default, CSharpCodeGenerationOptions.Default, new CSharpCodeGenerationService(document.Project.GetExtendedLanguageServices().LanguageServices), root.SyntaxTree.Options.LanguageVersion());
 #else
```

Comment:
so, this seems like something we likely need an analyzer for, right?  Like it's *farrrrrr* too easy to screw this up.  

Another option seems to be `(CSharpCodeGenarationService)document.GetRequiredLanguageService<ICodeGenerationService>()` (or just make CSharpCodeGenerationContextInfo take an ICodeGenService, i dont' think it needs the subtype.

Reply:
➡️ Updated BannedSymbols.txt to account for the new `Services` property, which was not present at the time `LanguageServices` property was excluded.